### ISSUE TITLE
Unhandled error for angular app creation

### ISSUE THREAD

**Issue by mellis481** [single-spa/create-single-spa#97]
I'm attempting to run the following CLI command:  

`create-single-spa employee-details --framework angular`.  

I get the following error:

```
(node:16824) ExperimentalWarning: The fs.promises API is experimental
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn ng ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

If I change the framework to `react`, it works without issue.  If I change the framework to `vue`, it is successful, although I see a `(node:20380) ExperimentalWarning: The fs.promises API is experimental` warning.

I'm using create-single-spa@1.5.4.

**Comment by joeldenning** [comment]
This works for me locally - I checked with both create-single-spa@1.5.4 and 1.6.0. A few questions to help diagnose:

- Which OS? Windows is the one we sometimes forget, so there's a chance this could be a windows-specific problem.
- Do you have Angular CLI globalled installed, or not? create-single-spa behaves differently depending on whether it is installed
- Which version of node?

The error message indicates that create-single-spa believes Angular CLI to be installed globally, but that when it attempted to invoke `ng new` that the operating system said that `ng` didn't exist.

**Comment by joeldenning** [comment]
I just tried this on a windows machine and saw the error - it appears to be windows specific. I'll try to diagnose

**Comment by joeldenning** [comment]
Hmm I fixed this simply by uninstalling create-single-spa and reinstalling it. Your installation of create-single-spa needs to be using generator-single-spa@1.5.4. We fixed this bug in #92.

Can you try uninstalling create-single-spa and reinstalling it?

**Comment by filoxo** [comment]
Closing due to inactivity but feel free to reopen if this is still an issue.
